src/code_style.md

```diff
@@ -0,0 +1,37 @@
+# Clear Code  
```

Comment:
Let's also discuss this on wednesday; there is some duplication with the developer guide. 

There is also a more general point, that asking someone to read 4 pages of contribution detail, seems quite overwhelming :<